main.js

```diff
@@ -92,7 +91,7 @@ view.drawMaze(maze, drawArrow, highlightOrigin, hideText);
 function mainLoop() {
     maze.iterate();
     view.drawMaze(maze, drawArrow, highlightOrigin, hideText);
-    if (animate) setTimeout(mainLoop, 1000 / animationFPS);
+    if (animate) requestAnimationFrame(mainLoop);
```

Comment:
Having a framerate it is not a bad idea. That guarantees the same speed on ony system. I suggest using deltaTime. with requestAnimationFrame.